build.gradle.kts

```diff
@@ -100,6 +100,8 @@ tasks.rat {
     "**/*.json",
 
     // Katas files
+    "learning/katas/**/course-info.yaml",
+    "learning/katas/**/task-info.yaml",
```

Comment:
IntelliJ edu course creation tool updates these files and removes the license.  Whenever one updates anything in the course files, they would have to manually update the license information.  These files only store metadata about the course.

Reply:
Makes sense! Lets not fight IntelliJ here, it's not a good use of anyone's time.